tasks/deploy/mainnet.ts

```diff
@@ -254,43 +254,84 @@ task("mainnet:siphon").setAction(async function (_: TaskArguments, hre) {
     await waitForTx(tx, debug, waitForBlocks);
 });
 
-task("mainnet:vestedEscrow:2").setAction(async function (_: TaskArguments, hre) {
-    const debug = true;
-    const waitForBlocks = 3;
-
-    const signer = await getSigner(hre);
-    const phase2 = await config.getPhase2(signer);
+task("mainnet:vestedEscrow:2")
```

Comment:
Might be worth encapsulating this into a script and then using it in the fork tests. This way we have a fork test that also covers our deployment script. Something similar to what we do with `deploySystem`